frontend/packages/console-app/src/components/cloud-shell/cloud-shell-utils.ts

```diff
@@ -50,6 +50,7 @@ export const CLOUD_SHELL_CREATOR_LABEL = 'controller.devfile.io/creator';
 export const CLOUD_SHELL_RESTRICTED_ANNOTATION = 'controller.devfile.io/restricted-access';
 export const CLOUD_SHELL_STOPPED_BY_ANNOTATION = 'controller.devfile.io/stopped-by';
 export const CLOUD_SHELL_PROTECTED_NAMESPACE = 'openshift-terminal';
+export const CLOUD_SHELL_SUBSCRIPTION_NAME = 'web-terminal';
```

Comment:
Is it used or it's just leftover from the previous way you figured out target namespace?

Reply:
It was just leftover